e2e/framework/testserver.go

```diff
@@ -16,7 +15,10 @@ import (
 	"testing"
 	"time"
 
-	"github.com/0xPolygon/minimal/command"
+	"github.com/0xPolygon/minimal/command/genesis"
+	ibft2 "github.com/0xPolygon/minimal/command/ibft"
```

Comment:
Please call it something among the lines of ibftCommand, not ibft2

Reply:
Got it 👍 
Fixed now